modules/account/controllers/profile.classic.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+* @package   test
+* @subpackage account
+* @author    test
+* @copyright 2019 Laurent Jouanneau and other contributors
+* @link      
+* @license    All rights reserved
+*/
+
+use Jelix\Authentication\Account\Manager;
+use Jelix\Authentication\Account\Account;
+
+class profileCtrl extends jController {
+
+    function index() {
+        $rep = $this->getResponse('html');
+
+        $currentUser = Manager::getCurrentUserAccount();
+        $tpl = new \jTpl();
+        if ($currentUser) {
+            $tpl->assign('user', $currentUser->getInfosArray());
+        }
+        $content = $tpl->fetch('profile_index');
+        $rep->body->assign('MAIN', $content);
+    
+        return $rep;
+    }
+
+    public function modify()
+    {
+        $rep = $this->getResponse('html');
+
+        $form = jForms::create('account~profile_modify');
```

Comment:
You forgot to fill the form with current values from the account object. And you should also try `jForms::get()` before creating a form because this method could be called after a redirection from the `save()` method. You should also add a parameter to `jForms::get()` and `jForms::create()`, which should be the id of the account. 

See https://github.com/jelix/jcommunity-module/blob/master/modules/jcommunity/controllers/account.classic.php  for an example of a similar controller.